assets/dev/js/editor/utils/presets-factory.js

```diff
@@ -249,6 +249,73 @@ presetsFactory = {
 			'66-33-33-66',
 		];
 	},
+
+	generateContainerGridPreset( preset ) {
+		const presets = {
+			'r2-c0': `
+				<svg width="92" height="47" viewBox="0 0 92 47" fill="none" xmlns="http://www.w3.org/2000/svg">
+					<rect x="91.2227" y="1.35059" width="44.5" height="90" transform="rotate(90 91.2227 1.35059)" fill="white" stroke="#515962" stroke-dasharray="3 3"/>
+					<path d="M91.0957 23.6006L1.34961 23.6006" stroke="#515962" stroke-dasharray="3 3"/>
+				</svg>
+			`,
+			'r0-c2': `
+				<svg width="92" height="46" viewBox="0 0 92 46" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I am thinking can you try to generate the svgs instead of 'hardcoding' them?

I am thinking of something like this: 

`<?php 

$image = get_field('image');
$svgImage = $image['url'];

if( !empty($image) ): ?>

    <?php echo file_get_contents($svgImage);?>

<?php endif; ?>`

I got this from Stackoverflow: https://stackoverflow.com/questions/54707632/output-svg-as-code-instead-of-image-tag-in-wordpress.

Reply:
@hein-obox I feel the current way is more performant than multiple `file_get_contents()` calls.

Reply:
Then we can leave the current code version.